.github/files/required-review.yaml

```diff
@@ -25,38 +18,43 @@
    - red
    - yamato-backup-and-security
 
-# Jetpack Approvers review the Jetpack plugin and all packages, except for those with specific team ownership.
-- name: Jetpack and packages
+# Jetpack Approvers need to review changes to the monorepo itself.
+- name: Monorepo itself
+  paths:
+   - '!projects/**'
+  teams:
+   - jetpack-approvers
+
+# Jetpack Approvers review the Jetpack plugin.
+- name: Jetpack-the-plugin
   paths:
-   - 'projects/packages/**'
    - 'projects/plugins/jetpack/**'
-   # Exclude packages managed by other teams, which are covered by other blocks below.
-   - '!projects/js-packages/image-guide/**'
-   - '!projects/js-packages/svelte-data-sync-client/**'
-   - '!projects/packages/wp-js-data-sync/**'
-   - '!projects/packages/backup/**'
-   - '!projects/packages/import/**'
-   - '!projects/packages/forms/**'
-   - '!projects/packages/search/**'
-   - '!projects/packages/stats/**'
-   - '!projects/packages/stats-admin/**'
-   - '!projects/packages/publicize/**'
-   - '!projects/packages/waf/**'
-   - '!projects/plugins/*/composer.lock'
-   - '!projects/plugins/*/composer.json'
-   - '!projects/plugins/*/changelog/*'
   teams:
    - jetpack-approvers
 
+# Packages owned by Ground Control.
+- name: Ground Control packages
+  paths:
+   - 'projects/js-packages/babel-plugin-replace-textdomain/**'
+   - 'projects/js-packages/eslint-changed/**'
+   - 'projects/js-packages/eslint-config-target-es/**'
+   - 'projects/js-packages/i18n-check-webpack-plugin/**'
+   - 'projects/js-packages/i18n-loader-webpack-plugin/**'
+   - 'projects/js-packages/remove-asset-webpack-plugin/**'
+   - 'projects/js-packages/webpack-config/**'
+   - 'projects/packages/autoloader/**'
+   - 'projects/packages/changelogger/**'
+   - 'projects/packages/codesniffer/**'
+   - 'projects/packages/ignorefile/**'
+   - 'projects/packages/phpcs-filter/**'
+  teams:
+   - jetpack-approvers
```

Comment:
I do not fully understand that part. Given that the Approvers group already is the fallback, why do we need to add those specific use-cases?

Reply:
We don't need to, at least not now. OTOH, this clearly documents that these packages are specifically intended to be owned, and keeps them that way should we decide to change the default for some reason.